quantum/keymap_extras/keymap_bepo.h

```diff
@@ -316,8 +317,7 @@
 #define BP_U_GRAVE BP_UGRV
 #define BP_DEAD_TREMA BP_DIAE
 #define BP_DTRM BP_DIAE
-#define BP_TYPOGRAPHICAL_APOSTROPHE BP_COMM
-#define BP_TAPO BP_COMM
+#define BP_TYPOGRAPHICAL_APOSTROPHE BP_TAPO
```

Comment:
```suggestion
#define BP_TYPOGRAPHICAL_APOSTROPHE BP_COMM
#define BP_TAPO BP_COMM
```
These defines are deprecated, and will be removed anyway. There's not much point in changing them to be "correct".